### DISCUSSION TITLE
ScriptReference/SceneManagement.SceneManager.GetSceneByBuildIndex

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#53] [Notes]
# ScriptReference/SceneManagement.SceneManager.GetSceneByBuildIndex

https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.GetSceneByBuildIndex.html

**Comment by vertxxyz** [comment]
For getting unloaded build scenes in the **editor** see: [`EditorBuildSettings.scenes`](https://docs.unity3d.com/ScriptReference/EditorBuildSettings-scenes.html) and [`SceneUtility.GetScenePathByBuildIndex`](https://docs.unity3d.com/ScriptReference/SceneManagement.SceneUtility.GetScenePathByBuildIndex.html).
